suite_cam7.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="dadadj" version="1.0">
+  <group name="physics_before_coupler">
+    <scheme>dadadj</scheme>
```

Comment:
I might add a comment here stating that this is dry adiabatic adjustment:

```suggestion
    <!-- Dry Adiabatic Adjustment -->
    <scheme>dadadj</scheme>
```

Reply:
Done.